RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -335,6 +335,12 @@ class PumpOpsSynchronous {
 
         var events = [TimestampedHistoryEvent]()
         var timeAdjustmentInterval: NSTimeInterval = 0
+        
+        // Going to scan backwards in time through events, so event time should be monotonically decreasing
+        // Start with some time in the future, to account for the condition when the pump's clock is ahead
+        // of ours by a small amount.
+        var timeCursor = NSDate(timeIntervalSinceNow: NSTimeInterval(minutes: 60))
+        let eventTimestampDeltaAllowance = NSTimeInterval(minutes: 60)
```

Comment:
I though so, but they're different things.  Line 342 is to handle pump vs phone differences (might need to increase this?), and line 343 is for variability between pump events (maybe should decrease this?); I think it's possible for events to appear out of order, for those events that get updated after the fact, like square wave boluses.